extensions/tracing/README.md

```diff
@@ -0,0 +1,52 @@
+# @loopback/extension-tracing
+
+This module contains a component to report tracing status using
+[Jaeger Tracing](https://github.com/jaegertracing/jaeger-client-node).
+
+## Stability: :warning:Experimental:warning:
+
+> Experimental packages provide early access to advanced or experimental
+> functionality to get community feedback. Such modules are published to npm
+> using `0.x.y` versions. Their APIs and functionality may be subject to
+> breaking changes in future releases.
+
+## Installation
+
+```sh
+npm install --save @loopback/extension-tracing
+```
+
+## Basic use
```

Comment:
Very cool and complicated stuff.

The `Basic Use` section should provide a description of what default options this component is using, and how it will affect the tracing. Perhaps an example of tracing one request and the output that appears on console or in a log.